### DISCUSSION TITLE
Guest difficulty update request feature

### DISCUSSION THREAD

**Discussion by lein-hub** [ppy/osu#25662] [Ideas] [answered]
originally written from [here](https://osu.ppy.sh/community/forums/topics/1852658?n=1)

Utilize the beatmap ownership feature, which allows you to update the difficulty you own even if you are not the owner of the beatmap.

When the guest difficulty is updated by the owner, the beatmap owner is notified and can update their local guest difficulty.

If this feature is technically difficult to implement, at least make the beatmap owner available for download the updated guest difficulty .osu file via osu homepage.

**Comment by peppy** [comment, marked as answer]
I tried to make this work in the past but it requires a rewrite of the system. Will come in the future.
